package.json

```diff
@@ -49,5 +51,8 @@
     "typescript": "^4.9.5",
     "update-ts-references": "^2.6.1"
   },
+  "resolutions": {
+    "@testing-library/dom": "^9.0.0"
```

Comment:
`@storybook/testing-library` brings in an older version of `@testing-library/dom`. Unfortunately, multiple versions of this package causes annoying `act` warnings in tests.